src/textbox.cpp

```diff
@@ -151,7 +151,7 @@ void TextBox::draw(NVGcontext* ctx) {
         spinArrowsWidth = 14.f;
 
         nvgFontFace(ctx, "icons");
-        nvgFontSize(ctx, ((mFontSize < 0) ? mTheme->mButtonFontSize : mFontSize) * 1.2f);
+        nvgFontSize(ctx, ((mFontSize < 0) ? mTheme->mButtonFontSize : mFontSize) * detail::__icon_scale);
```

Comment:
Seventh icon scale.  This one is not the "default" `1.5`, and may also need a different value?

Reply:
Yep, the little scroll handles got bigger. It looked nicer before.